data/presets/highway/path/traffic_island_shared.json

```diff
@@ -0,0 +1,31 @@
+{
+    "locationSet": {
+        "include": [
+            "fr",
+            "lt",
+            "pl",
+            "de",
+            "il",
+            "ps"
+        ]
+    },
+    "icon": "temaki-pedestrian_and_cyclist",
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "path": "traffic_island"
+    },
+    "addTags": {
+        "highway": "path",
+        "path": "traffic_island"
+    },
+    "reference": {
+        "key": "path",
+        "value": "traffic_island"
```

Comment:
We should add this wiki page as a stub at least…

Reply:
done. for some reason the wiki UI for linking data items to pages is completely broken, but I think i've managed it...